iceoryx2-bb/cxx/tests/BUILD.bazel

```
@@ -22,9 +22,10 @@ cc_test(
     srcs = glob([
         "src/**/*.cpp",
         "src/**/*.hpp",
-    ]),
+        ]),
```

By adding the inl files to `src` the compilation error for the tests is fixed.

```suggestion
    srcs = glob([
        "src/**/*.cpp",
        "src/**/*.hpp",
        "src/import/testing/include/**/*.inl",
        ]),
```